pkg/types/powervs/validation/platform_test.go

```diff
@@ -2,7 +2,7 @@ package validation
 
 import (
 	"crypto/rand"
-	"errors"
```

Comment:
No need to remove.

Reply:
This one, I put back and it compiled fine.

Reply:
But lint complains:
```
pkg/types/powervs/validation/platform_test.go:5:2: "errors" imported but not used (typecheck)
	"errors"
```

Reply:
Yes because you changed the line that used it below: `errors.New(...)`. I was saying that those `errors.New` calls don't necessarily need to be replaced because they are using the built-in `errors` lib, not the archived `github.com/pkg/errors` package one.